### ISSUE TITLE
Multiple file extensions

### ISSUE THREAD

**Issue by craigburke** [bertramdev/asset-pipeline-core#9]
Hard to come up with a good title for this one, but this is an issue I ran into with my angular template module: https://github.com/craigburke/angular-template-asset-pipeline

This is related to the issue of compound file extensions like `tpl.html.` These files ultimate resolve to JS files, but it doesn't work to use require_tree within a JS. 

At least for the FileSystemResolver, I tracked down the issue here:
https://github.com/bertramdev/asset-pipeline-core/blob/master/asset-pipeline-core/src/main/groovy/asset/pipeline/fs/FileSystemAssetResolver.groovy#L134

Basically with these files, the mime type ends up being `text/html` instead of `application/javascript` so they won't be included.  

I can work up a pull request for a change to the AssetHelper class so that all potential mime types would be returned from assetMimeTypeForURI, but I wanted to get your opinion before proceeding.

**Comment by davydotcom** [comment]
support was added in 2.2.0 enjoy

**Comment by craigburke** [comment]
Thanks, dude! :+1:
